packages/web/components/Player/Section/PlayerMemberships.tsx

```diff
@@ -135,6 +136,45 @@ const DAOListing: React.FC<DAOListingProps> = ({
   );
 };
 
+export const DAOMembershipSmall: React.FC<DAOListingProps> = ({
+  membership: { title, chain, address, logoURL, guildname },
+}) => {
+  const daoURL = useMemo(() => getDAOLink(chain, address), [chain, address]);
+
+  return (
+    <Tooltip label={title ?? `Unknown ${chain} DAO`}>
+      <Flex
+        onClick={(e) => {
+          e.preventDefault();
+          if (guildname != null) {
+            window.location.href = guildname;
```

Comment:
The Next way to do this is using [`useRouter`](https://nextjs.org/docs/api-reference/next/router).

```suggestion
            router.push(guildname);
```